Homework-№4-from-28.09.2022/task 2/task2.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+bool checker(int *firstArray, int *secondArray);
+// this function will verify for a match two arrays
+// and return the result of this verifying
+
+void tests(void);

```

Comment:
```suggestion
bool tests(void);
```
Тесты возвращают bool, чтобы в main можно было определить, пройдены тесты или нет.